src/interactivity/onecore/ConIoSrvComm.cpp

```diff
@@ -24,7 +24,7 @@ extern void UnlockConsole();
 using namespace Microsoft::Console::Interactivity::OneCore;
 
 ConIoSrvComm::ConIoSrvComm() :
-    _inputPipeThreadHandle(INVALID_HANDLE_VALUE),
+    _inputPipeThreadHandle(nullptr),
     _pipeReadHandle(INVALID_HANDLE_VALUE),
     _pipeWriteHandle(INVALID_HANDLE_VALUE),
     _alpcClientCommunicationPort(INVALID_HANDLE_VALUE),

```

Comment:
I left the pipe handles alone because pipe handles in the Windows API tend to be treated like file handles, so while I was unsure, I thought INVALID_HANDLE_VALUE would work for pipes just as it does for files.

I had no clue about the RPC (ALPC) port handle, so I left that alone too.